### DISCUSSION TITLE
Lot level information

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#665] [Q&A] [answered]
In our application we can create notices without lots, looking at the notice.json, there is data that is always included in the lots part, but in this case it would belong to us in the notice. 
For example, the AwardCriteria, in our case they can be at the notice level (notice without lots), but in the eforms they would only be sent within a lot. Would it be necessary to always create a lot to send the eform?

Thanks in advance,
Alex

**Comment by AnnaKSZ** [comment, marked as answer]
Hi Alexa,

https://docs.ted.europa.eu/eforms/latest/schema/procedure-lot-part-information.html
"For Procedure without Lots (i.e. with a single "technical" lot), the ID shall be LOT-0000."

If there is no Lots of the Procedure you have to make a technical lot with "LOT-0000" id.

Regards
Anna

**Reply by mdewinne** [reply]
I believe the requirement to use LOT-0000 as ID for a single (technical) lot has been dropped. If there is only one lot, you can just as well use LOT-0001. 

This is how we have implemented it in our national tendering platform and it makes lot management a lot easier, especially if tenders are still in draft or pre-competition phase and the user is still allowed to change the number of lots (e.g. go from one to multiple lots or vice versa).

It would be good if OP could confirm this and update the documentation accordingly.

**Reply by YvesJo** [reply]
Hi,
This constraint on the technical lot identifier has currently been waived. It is foreseen to have it again in the future, also it is good to follow the recommendation. We understand that this may be challenging for some specific cases, and until the rule gets activated either approach will be accepted.
When a procedure is not subdivided into lots, there shall however be a "technical" lot which will contain the details about the procedure, including Award Criteria information (there is no field defined for this at the procedure level).
